themes/default/layouts/product/esc.html

```diff
@@ -40,8 +40,8 @@ <h3>Centralized environments, secrets, and configurations management for cloud a
         </div>
     </section>-->
     <section id="overview" class="max-w-5xl mx-auto text-center my-16 px-6">
-            <h3>Centralized environments, secrets, and configurations management for cloud applications and infrastructure</h3>
-            <p class="text-center">Today’s cloud environments access a multitude of configurations – including network settings, deployment options, API Keys, and other important secrets, such as database credentials –  from many different types of cloud infrastructure and SaaS services. Every team stores these configurations in different locations, from different secrets managers to plaintext files. This sprawl leads to configurations that are untraceable and lack access controls, leading to operational bottlenecks, outages linked to human error, and security breaches. Pulumi ESC enables you to centrally manage all configurations and secrets across your organization.</p>
+            <h3>Centralized environments, secrets, and configuration management for cloud applications and infrastructure</h3>
+            <p class="text-center">Today’s cloud environments access a multitude of configurations – including network settings, deployment options, API Keys, and other important secrets, such as database credentials –  from many different types of cloud infrastructure and SaaS services. Every team stores these configuration in different locations, from different secrets managers to plaintext files. This sprawl leads to configuration that are untraceable and lack access controls, leading to operational bottlenecks, outages linked to human error, and security breaches. Pulumi ESC enables you to centrally manage all configuration and secrets across your organization.</p>
```

Comment:
```suggestion
            <p class="text-center">Today’s cloud environments access a multitude of configurations – including network settings, deployment options, API Keys, and other important secrets like database credentials –  from many different types of cloud infrastructure and SaaS services. Every team stores configuration settings like these in different locations, from secrets managers to plaintext files. This sprawl leads to configuration that is uncontrolled and untraceable, leading to operational bottlenecks, outages linked to human error, and security breaches. Pulumi ESC enables you to centrally manage all configuration and secrets across your organization.</p>
```

Curious why this is duplicated, though; it'll be easy to change it in one place and forget to change it in the other.